### PR TITLE
Move iOS client from MRC to ARC

### PR DESCRIPTION
**Pull request by oferDavidyan**

### CODE REVIEW

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Comment by bmiklautz** [comment]
@oferDavidyan thanks for the pull request. However the commit messages are not really saying much  could you either make one commit with a proper message or update the commit messages?

Also it seems that it doesn't built on ios cleanly https://ci.freerdp.com/job/ios/998/console

File: scripts/OpenSSL-DownloadAndBuild.command [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 ## Settings
 # openssl version to use
-OPENSSLVERSION="1.0.2a"
-MD5SUM="a06c547dac9044161a477211049f60ef"
+OPENSSLVERSION="1.0.2f"
+MD5SUM="b3bf73f507172be9292ea2a8c28b659d"
```
Comment (akallabeth) [inline scripts/OpenSSL-DownloadAndBuild.command]: OpenSSL.org does no longer supply MD5 sums, could you change that to SHA256? Needs some adjustment below when checking the hash.

**Comment by bmiklautz** [comment]
@xproax thanks that you are interested in FreeRDP but can you please stop approving pull requests (multiple times)!

**Comment by akallabeth** [comment]
@oferDavidyan I'll close this pull since you did not react to change requests and it needs a rebase. If still interested in getting this merged please reopen/create a new one.

**Pull request closed without merging**